### PR TITLE
Update behat instructions in README.md

### PR DESCRIPTION
**Pull request by simoncoggins**
Previous version doesn't mention how to configure to use selenium hub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,6 +307,13 @@ vendor/bin/phpunit
 
 Make sure your config file contains the Behat configuration needed and the database is ready.
 
+If you want to run selenium tests make sure the selenium container is started:
+```bash
+tup selenium-hub
```
Comment (elkbullwinkle) [inline README.md]: Shouldn't you also start nodes?

Comment (simoncoggins) [reply README.md]: Apparently it happens automatically as selenium-hub has been marked as dependent on nodes.

Comment (elkbullwinkle) [reply README.md]: Oh nice, just for reference you can scale the number of nodes you can --scale and a number if you need more than one node

File: README.md [hunk 2]
```diff
@@ -307,6 +307,13 @@ vendor/bin/phpunit
...
+```
+
+and you have a behat_local.yml file in your root code folder that provides the correct configuration (e.g. points to the selenium-hub host and the base_url is correct).
```
Comment (derschatta) [inline README.md]: this actually depends on the version. behat_local.yml is not supported in older versions (I think 9 and older), there you need to have something like

```
$CFG->behat_profiles = [
    'default' => [
        'browser' => 'chrome',
        'extensions' => [
            'Behat\MinkExtension' => [
                'selenium2' => [
                    'browser' => 'chrome',
                ]
            ]
        ]
    ]
];
```
in your config.php

There are also other configuration options which need to be set, maybe adding an example here would make it easier for others (we could also create a separate BEHAT.md file with more details around it if this file gets too big.

Example configuration (Which should work for old and new versions I think)
```
$CFG->behat_wwwroot = 'http://totara73.behat/13';
$CFG->behat_prefix = 'bht_';
$CFG->behat_dataroot = '/var/www/totara/data/ver13.pgsql.behat';

$CFG->behat_profiles = [
    'default' => [
        'browser' => 'chrome',
        'extensions' => [
            'Behat\MinkExtension' => [
                'selenium2' => [
                    'browser' => 'chrome',
                ]
            ]
        ]
    ]
];

$CFG->behat_config = array(
    'default' => array(
        'extensions' => array(
            'Behat\MinkExtension\Extension' => array(
                'browser_name' => 'chrome',
                'default_session' => 'selenium2',
                'selenium2' => array(
                    'browser' => 'chrome',
                    'wd_host' => 'http://selenium-hub:4444/wd/hub',
                    'capabilities' => array(
                        "browser" => "chrome",
                        "browserName" => "chrome",
                        "browserVersion" => "",
                        "version" => '',
                        'platform' => 'LINUX'
                    )
                )
            )
        )
    ),
);
```

**Pull request merged**